source/system-administrators/upgrade/upgrading-to-craftercms-3-0.rst

```diff
@@ -44,27 +44,25 @@ Export the 2.5.x Site
 Export the site's Studio config (``cstudio/config/sites/{siteName}``) and the site's content (``wem_projects/{siteName}/{siteName}/work-area`` folders
 from Alfresco or from your local 2.5.x repository. Then, copy these folders into a location in the system with Crafter 3.0.x installed.
 
-^^^^^^^^^^^^^^^^^^^^^
-Run the Import Script
-^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^
+Run the Migration Script
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Before running the script, make sure:
+Before running the script, make sure:
 
-   - ``curl`` and ``git`` are installed.
-   - Studio 3.0.x is running.
-   - There's no site in Studio with the same name as the one you're migrating.
+  - ``curl`` and ``git`` are installed.
 
-#. Run the ``migrate.sh``, which is located under ``crafter-authoring/bin/migration`` directory. The script takes 3 parameters:
+Run the ``migrate.sh``, which is located under ``crafter-authoring/bin/migration`` directory. The script takes 3 parameters:
 
-   - The name of the new 3.0 site where the original site will be migrated.
-   - The location of the 2.5 Studio configuration (where the content-types reside).
-   - The root of the 2.5 site (where the site, scripts, static-assets and template folders reside).
+  - The name of the new 3.0 site where the original site will be migrated.
+  - The location of the 2.5 Studio configuration (where the content-types reside).
+  - The root of the 2.5 site (where the site, scripts, static-assets and template folders reside).
```

Comment:
It would be nice to include a sample of the command with parameters after this line.  Something like:

.. code-block:: sh
    ./migrate.sh newSite /path/to/content/types /root/of/old/site